yt_dlp/extractor/microsoftembed.py

```diff
@@ -63,3 +71,217 @@ def _real_extract(self, url):
+
+        return self._sub_to_dict(captions)
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        ism_url = self._search_regex(r'StreamUrl\s*=\s*"([^"]+manifest)"', webpage, 'ism url')
+
+        return {
+            'id': video_id,
+            'title': self._og_search_title(webpage),
+            'description': self._og_search_description(webpage),
+            'formats': self._extract_ism(ism_url, video_id),
+            'thumbnail': self._og_search_thumbnail(webpage),
+            'subtitles': self._extract_subtitle(webpage, video_id),
+        }
+
+
+class MicrosoftLearnIE(MicrosoftMediusBaseIE):
+    _VALID_URL = r'https?://learn\.microsoft\.com/(?:[\w\-]+/)?(?P<type>events|shows)/(?P<series>[\w\-]+)(?:/(?P<id>[^?#/]+))?'
+
+    _TESTS = [{
+        'url': 'https://learn.microsoft.com/en-us/events/build-2022/ts01-rapidly-code-test-ship-from-secure-cloud-developer-environments',
+        'info_dict': {
+            'id': '9640d86c-f513-4889-959e-5dace86e7d2b',
+            'ext': 'ismv',
+            'title': 'Rapidly code, test and ship from secure cloud developer environments - Events',
+            'description': 'md5:f26c1a85d41c1cffd27a0279254a25c3',
+            'timestamp': 1653408600,
+            'upload_date': '20220524',
+            'thumbnail': r're:https://mediusimg\.event\.microsoft\.com/video-\d+/thumbnail\.jpg.*',
+        },
+    }, {
+        'url': 'https://learn.microsoft.com/en-us/events/build-2022',
+        'info_dict': {
+            'id': 'build-2022',
+            'title': 'Microsoft Build 2022 - Events',
+            'description': 'md5:c16b43848027df837b22c6fbac7648d3',
+        },
+        'playlist_count': 201,
+    }, {
+        'url': 'https://learn.microsoft.com/en-us/shows/bash-for-beginners/what-is-the-difference-between-a-terminal-and-a-shell-2-of-20-bash-for-beginners/',
+        'info_dict': {
+            'id': 'd44e1a03-a0e5-45c2-9496-5c9fa08dc94c',
+            'ext': 'ismv',
+            'title': 'What is the Difference Between a Terminal and a Shell? (Part 2 of 20)',
+            'description': 'md5:7bbbfb593d21c2cf2babc3715ade6b88',
+            'timestamp': 1676339547,
+            'upload_date': '20230214',
+            'thumbnail': r're:https://learn\.microsoft\.com/video/media/.*\.png',
+            'subtitles': 'count:14',
+        },
+        'params': {'listsubtitles': True},
+    }, {
+        'url': 'https://learn.microsoft.com/en-us/shows/bash-for-beginners',
+        'info_dict': {
+            'id': 'bash-for-beginners',
+            'title': 'Bash for Beginners',
+            'description': 'md5:16a91c07222117d1e00912f0dbc02c2c',
+        },
+        'playlist_count': 20,
+    }]
+
+    def _entries(self, url_base, video_id):
+        skip = 0
+        while True:
+            playlist_info = self._download_json(f'{url_base}&$skip={skip}', video_id, f'Downloading entries {skip}')
```

Comment:
```suggestion
            playlist_info = self._download_json(url_base, video_id, f'Downloading entries {skip}', query={
                'locale': 'en-us',
                '$skip': skip,
            })
```
This will update the query. I think its better to have the `en-us` here, especially since the url-result created from it in this function also hardcodes `en-us`